pkgs/development/compilers/nim/nimble-versions.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/src/nimblepkg/download.nim b/src/nimblepkg/download.nim
+index 5750d90..76676bd 100644
+--- a/src/nimblepkg/download.nim
++++ b/src/nimblepkg/download.nim
+@@ -259,16 +259,6 @@ proc downloadPkg*(url: string, verRange: VersionRange,
+     doDownload(modUrl, downloadDir, verRange, downMethod, options)
+   )
+ 
+-  if verRange.kind != verSpecial:
```

Comment:
Should this be conditional on the fact that we are running in a Nix build? I could imagine regular CLI users might want to retain that check.